features/dd-sdk-android-session-replay/src/test/kotlin/com/datadog/android/sessionreplay/forge/WireframeBoundsForgeryFactory.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.forge
+
+import com.datadog.android.sessionreplay.internal.processor.WireframeBounds
+import fr.xgouchet.elmyr.Forge
+import fr.xgouchet.elmyr.ForgeryFactory
+
+internal class WireframeBoundsForgeryFactory : ForgeryFactory<WireframeBounds> {
+    override fun getForgery(forge: Forge): WireframeBounds {
+        val left = forge.aLong(min = 0, max = 1000)
+        val right = left + forge.aLong(min = 2, max = 1000)
+        val top = forge.aLong(min = 0, max = 1000)
+        val bottom = top + forge.aLong(min = 2, max = 1000)
```

Comment:
just curious: why min value is 2 and not 1? Also does max value matter?

Reply:
@mariusc83 do you remember why these values? its from the commit that you added

Reply:
the max value matters yes as we want to avoid Long overflows in our tests. I do not remember why I specifically added min = 2 instead of 1. I should have added a comment there 🤦 . @jonathanmos maybe you can change it to 1 and run the tests multiple times in the `WireframeUtilsTest` . I am pretty sure you will find the reason why.

Reply:
It might not be a relevant reason so in that case you can switch it to 1.

Reply:
thanks - I wasn't able to reproduce any flakiness with multiple runs so I'll set to 1.